app/experimenter/experiments/models.py

```diff
@@ -595,10 +601,13 @@ def bucket_namespace(self):
             feature_config.slug
             for feature_config in self.feature_configs.all().order_by("slug")
         )
+
         if self.channel:
             keys.append(self.channel)
 
         if self.is_rollout:
+            if self.targeting_config_slug:
+                keys.append(self.targeting_config_slug.replace("_", "-"))
```

Comment:
Why replace?

Reply:
Just for consistency. The rest of the bucketing is `xxx-xxxx-xxx-xxx` so it looked kinda wonky with `xx-xxx-xxx-xxx_xxxx_xxxx`

Reply:
Right but it's not actually necessary for the behaviour and logic, and particularly since if you look at it and go 'where did this part come from' and it's not actually what's in the targeting config, that would be confusing.  I wouldn't do this.